addons/web_editor/static/lib/odoo-editor/src/base_style.css

```diff
@@ -0,0 +1,3 @@
+li.oe-nested {
+    display: block;
+}
```

Comment:
Missing empty line at end of file, this can be seen pretty easily in github ;-)

Reply:
fixed